lte/gateway/python/magma/pipelined/app/inout.py

```diff
@@ -401,8 +401,22 @@ def _setup_non_nat_monitoring(self, datapath):
         threading.Event().wait(1)
 
 
-def _install_vlan_egress_flows(dp, table_no, out_port, ip,
+def _install_vlan_egress_flows(dp, table_no, ip, out_port=None,
                                priority=0, direction=Direction.IN):
+    """
+    Install egress flows
+    :param dp datapath
```

Comment:
We should stick to 1 docstring style across the pipelined codebase, can you refactor it to be similar to
```
        """
        Activate the flows for a subscriber based on the rules stored in Redis.
        During activation, a default flow may be installed for the subscriber.

        Args:
            imsi (string): subscriber id
            ip_addr (string): subscriber session ipv4 address
            static_rule_ids (string []): list of static rules to activate
            dynamic_rules (PolicyRule []): list of dynamic rules to activate
        """
```